sdk/storage/storage-blob/MigrationGuide.md

```diff
@@ -0,0 +1,263 @@
+# Guide for migrating to `@azure/storage-blob` from `azure-storage`
```

Comment:
Should we specify v12 here because there is v10 for the same package and people might get confused?

Reply:
I'm trying to add version info here. 
What do you think about: 
     Guide for migrating to `@azure/storage-blob` v12 from `azure-storage`
or
    Guide for migrating to `@azure/storage-blob` with version 12.*.* from `azure-storage`

Reply:
My vote is for the first option